keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -198,16 +283,48 @@ Feature is already in Beta.
 
 #### Beta -> GA Graduation
 
-- 3 examples of real world usage
-- support for remote TLS handshakes (e.g. TPM/KMS-hosted keys)
+- Three examples of real world usage
+  + Confirm interactive and non-interactive UX is acceptable
+  + Confirm no hacks are being performed to workaround limitations
+  + Confirm that configuration of plugin
+    * Is correctly handled
+    * Is well-supported by the `kubeconfig` file format
+- Create the `client.authentication.k8s.io/v1` `ExecCredential` struct
+- Address known bugs and add tests to prevent regressions
+- Docs are up-to-date with latest version of APIs
+- Docs describe set of best practices (i.e. do not mutate `kubeconfig`)
+
+Question: does this need conformance tests?  What would such a test look like?
```

Comment:
Note to self: reach out to conformance working group.  We are aiming for good test coverage for GA and not conformance, but it may be good to have a discussion around this.